### ISSUE TITLE
custom hover tooltips should be shown on keyboard focus

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#199228]
We cannot show native tooltips for icon labels unless we enable a chromium flag. Setting such a flag results in both native and custom hovers showing up on mouse over. 

As a result, for now, we can only show the tooltip on keyboard focus for custom hovers. 

Verify that for icon labels with custom hovers, for example, the editor action ones, the tooltip shows up on keyboard focus of the item. 

Enable `Tab moves focus mode` with `ctrl+shift+m` to test this and use arrow keys to navigate in toolbars.

![Screenshot 2023-11-27 at 2 36 14 PM](https://github.com/microsoft/vscode/assets/29464607/9320d2a9-c0eb-4393-8cea-1776d8cf2e62)
